Resources/public/css/views/universaldiscovery/search.css

```diff
@@ -128,3 +129,12 @@
 .ez-view-universaldiscoverysearchview .ez-searchresult-navigation-link[rel="prev"] {
     padding-right: 1em;
 }
+
+.ez-view-universaldiscoverysearchview .ez-searchresult-content {
+    overflow: auto;
+    height: calc(100% - 6em);
```

Comment:
is there a way to avoid such *magic* value ? I understand `6em` comes from the height of `.ez-ud-pane-search-form` (which should not be set btw) but `9em` below, I don't get how you get this.